content/tracing/visualization/_index.md

```diff
@@ -0,0 +1,95 @@
+For instance, a simple web application may consist of two services: 
+
+* A single `webapp` service and a single `database` service
+
+While a more complex environment may break it out into 6 services: 
+
+* 3 separate services: `webapp`, `admin`, and `query`.
+* 3 separate external service:  `master-db`,  `replica-db`, and `yelp-api`.
+
+APM automatically assigns names to your services; however you can also name them explicitly. See instructions for: [Go](https://godoc.org/github.com/DataDog/dd-trace-go/tracer#Service), [Java](/tracing/setup/java/#configuration), [Python](http://pypi.datadoghq.com/trace/docs/#get-started), [Ruby](http://www.rubydoc.info/gems/ddtrace/).
+
+Service names:
+
+* **must be lowercase, alphanumeric characters**.
+* **cannot have more than 50 characters**.
+* **cannot contain spaces** (spaces are replaced with underscores).
+* must adhere to [metric naming rules](/developers/metrics/).
+
+**Note**: Service must have a type attached, APM automatically assigns services one of four types: web, database, cache, custom.
+
+You can also [alert](/monitors/monitor_types/apm/) on any service level metric. Read more about monitoring services in APM on the [service list](/tracing/visualization/services_list/) and [service dashboard](/tracing/visualization/service) pages.
+
+## Resources
+
+**A Resource is particular action for a service**.  
+
+* **For a web application**: some examples might be a canonical URL, such as `/user/home` or a handler function like `web.user.home` (often referred to as “routes” in MVC frameworks).
+* **For a SQL database**: a resource is be the query itself, such as `SELECT * FROM users WHERE id = ?`.
+
+Resources should be grouped together under a canonical name, like `/user/home` rather than have `/user/home?id=100` and `/user/home?id=200` as separate resources. APM automatically assigns names to your resources; however you can also name them explicitly. See instructions for: [Go](/tracing/setup/go/), [Java](/tracing/setup/java/#configuration), [Python](/tracing/setup/python/), [Ruby](/tracing/setup/ruby/).
+
+These resources can be found after clicking on a particular [service](/tracing/visualization/services_list/#resource).
+
+Resource names: 
+
+* **must be lowercase, alphanumeric characters**
+* **cannot exceed 5000 bytes**
+
+[Alert](/tracing/faq/how-to-create-a-monitor-over-every-resource-apm) on any resource level metric. Read more about monitoring resources in APM on the [resource dashboard](/tracing/visualization/resource) page.
+
+### Resource Cardinality
+
+When a resource such as URL or SQL query cannot be aggregated, it significantly increases the cardinality of resources (ie. the number of unique aggregate resources) to be stored in Datadog.  
+
+Having a very high cardinality of resources makes Datadog less usable:
+
+* Lots of entry in the resource list is not optimal for navigation
+* Statistics are less relevant (as they are too fragmented)
```

Comment:
These two bullet points don't have periods, but the ones above on this page do.

Reply:
👌